ui/address/tokenSelect/TokenSelectItem.tsx

```diff
@@ -19,7 +19,7 @@ const TokenSelectItem = ({ data }: Props) => {
     switch (data.token.type) {
       case 'ERC-20': {
         const tokenDecimals = Number(data.token.decimals) || 18;
-        const text = `${ BigNumber(data.value).dividedBy(10 ** tokenDecimals).toFormat(2) } ${ data.token.symbol || '' }`;
+        const text = `${ BigNumber(data.value).dividedBy(10 ** tokenDecimals).dp(8).toFormat() } ${ data.token.symbol || '' }`;
```

Comment:
probably, the test suite should be modified as well to show the case with 8+ digits after the comma

Reply:
I think it is more about we chose the correct way to display these values and it doesn't affect anything in the layout